tests/tests/operation_tests.cpp

```diff
@@ -7423,7 +7424,450 @@ BOOST_FIXTURE_TEST_SUITE(auction_window_tests, votes_extended_fixture)
+
+            // Needed to be sured, that auction window's been already enabled.
+            generate_blocks(fc::time_point_sec(STEEMIT_HARDFORK_0_6_REVERSE_AUCTION_TIME), true);
+
+            auto &wso = db->get_witness_schedule_object();
+            db->modify(wso, [&](witness_schedule_object &w) {
+                w.median_props.auction_window_size = 10 * 60;
+            });
+
+            generate_block();
+
+            ACTOR(alice);
+            int voters_count = 10;
+
+            comment_operation comment;
+            comment_options_operation cop;
+            signed_transaction tx;
+            set_price_feed(price(ASSET("1.000 GOLOS"), ASSET("1.000 GBG")));
+
+            BOOST_TEST_MESSAGE("Create a post.");
+            comment.author = "alice";
+            comment.permlink = "testauw2";
+            comment.parent_permlink = "test";
+            comment.title = "test";
+            comment.body = "Let's test auction window improvements!";
+
+            tx.operations.clear();
+            tx.signatures.clear();
+            GOLOS_CHECK_NO_THROW(push_tx_with_ops(tx, alice_private_key, comment));
+            generate_block();
+
+            protocol::comment_auction_window_reward_destination dest;
+            dest.destination = protocol::to_reward_fund;
+
+            cop.author = comment.author;
+            cop.permlink = comment.permlink;
+
+            cop.extensions.insert(dest);
+
+            tx.operations.clear();
+            tx.signatures.clear();
+
+            GOLOS_CHECK_NO_THROW(push_tx_with_ops(tx, alice_private_key, cop));
+            generate_block();
+            auto & alice_post = db->get<comment_object, by_permlink>(boost::make_tuple(comment.author, comment.permlink));
+            generate_voters(voters_count);
+
+            BOOST_TEST_MESSAGE("Create votes.");
+            BOOST_CHECK_EQUAL(alice_post.cashout_time, alice_post.created + STEEMIT_CASHOUT_WINDOW_SECONDS);
+
+            generate_blocks((alice_post.created + alice_post.auction_window_size), true);
+            vote_sequence(comment.author, comment.permlink, voters_count, 5);
+
+
+            comment_fund total_comment_fund(*db);
+            comment_reward alice_post_reward(*db, total_comment_fund, alice_post);
+            auto& gpo = db->get_dynamic_global_properties();
+
+            generate_blocks((alice_post.cashout_time), true);
+
+            share_type rwd = 0;
+            for (int i = 0; i < voters_count; i++) {
+                std::string acc_name = "voter" + std::to_string(i);
+                auto& account = db->get_account(acc_name);
+                rwd += account.curation_rewards;
+            }
+            auto& alice_acc = db->get_account(alice_post.author);
+
+            double auw_tokens = (gpo.total_reward_fund_steem.amount.value);
+            double total_payout = (alice_acc.posting_rewards.value) * 100.0 / 75.0;
+            double voters_reward = (rwd.value);
```

Comment:
Do not define useless variables.

Reply:
WTF? I do use them in calculations.

Reply:
@AKorpusenko You using values. Use values. But don't define variables for values which used just 1 time.
`double voters_reward = (rwd.value); // ...or if such value already defined:
`